rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "stable"
```

Comment:
We can also pin to a specific version, not sure what the preference is

Reply:
`stable` works for me. We have enough crazy build-time stuff happening in the bindings, last thing we need is to be doing that on a nightly build